src/wayland_client_core.zig

```diff
@@ -125,3 +125,39 @@ pub const EglWindow = opaque {
     extern fn wl_egl_window_get_attached_size(egl_window: *EglWindow, width: *c_int, height: *c_int) void;
     pub const getAttachedSize = wl_egl_window_get_attached_size;
 };
+
+pub const CursorTheme = opaque {
+    extern fn wl_cursor_theme_load(name: ?[*:0]const u8, size: c_int, shm: *client.wl.Shm) ?*CursorTheme;
```

Comment:
Let's make this return an `error.LoadThemeFailed` but keep `getCursor()` returning an optional.